conftest.py

```diff
@@ -0,0 +1,56 @@
+"""
+This file configures a local pytest plugin, which allows us to configure plugin hooks to control the
+execution of our tests. Either by loading in fixtures, configuring directories to ignore, etc
+
+Local plugins: https://docs.pytest.org/en/3.10.1/writing_plugins.html#local-conftest-plugins
+Hook reference: https://docs.pytest.org/en/3.10.1/reference.html#hook-reference
+"""
+import os
+import re
+import sys
+
+import pytest
+
+PY_DIR_PATTERN = re.compile(r'^py[23][0-9]$')
+
+
+# Determine if the folder should be ignored
+# https://docs.pytest.org/en/3.10.1/reference.html#_pytest.hookspec.pytest_ignore_collect
+# DEV: We can only ignore folders/modules, we cannot ignore individual files
+# TODO: Should this be `pytest_collect_directory`? or `pytest_collect_file`?
+# DEV: We must wrap with `@pytest.mark.hookwrapper` to inherit from default (e.g. honor `--ignore`)
+#      https://github.com/pytest-dev/pytest/issues/846#issuecomment-122129189
+@pytest.mark.hookwrapper
+def pytest_ignore_collect(path, config):
+    """
+    Skip directories defining a required minimum Python version
+
+    Example::
+
+        File: tests/contrib/vertica/py35/test.py
+        Python 2.7: Skip
+        Python 3.4: Skip
+        Python 3.5: Collect
+        Python 3.6: Collect
+
+    :rtype: bool
+    :returns: ``True`` to skip the directory, ``False`` to collect
```

Comment:
Unless I'm missing something, I don't see a `True` or `False` being returned by this function. Does it work?

Reply:
Whoops, good catch. Since wrapping it doesn't need to return anymore, just update `outcome.force_result()`